src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -114,7 +116,28 @@ public void setUpdateDate(Timestamp updateDate) {
     this.updateDate = updateDate;
   }
 
-  /**
+  public Integer getCountCollaborators() {
+    ArrayList<String> group = new ArrayList<>();
+    if (Objects.nonNull(getUserId())) {
+      group.add(getUserId().toString());
```

Comment:
I was talking to Johnathan and he said we are looking to include the researcher who created the DAR in the count of unique users

Reply:
If that's the case, then we definitely need to collect all the emails as a list and test for unique counts on that. The naming is also confusing here since it's no longer `Collaborators` but any associated user to the DAR. We should name this to be more reflective of what it really is.

Reply:
Okay, good idea, where could we find the email for the researcher?